onprc_ehr/src/org/labkey/onprc_ehr/ONPRC_EHRManager.java

```diff
@@ -90,6 +91,8 @@ public class ONPRC_EHRManager
     @Queryable
     public static final String CAGE_MEDICAL_EXEMPTION_FLAG = "Medical";
 
+    private String _caseSequence = "org.labkey.onprc_ehr.cases";
```

Comment:
Consider switching to private static final and using all-caps for the variable name

Reply:
updated